frontend/root-config/src/index.ejs

```diff
@@ -53,7 +53,8 @@
         "imports": {
           "single-spa": "https://cdn.jsdelivr.net/npm/single-spa@5.9.0/lib/system/single-spa.min.js",
           "vue": "https://cdn.jsdelivr.net/npm/vue@2.6.11/dist/vue.min.js",
-          "vue-router": "https://cdn.jsdelivr.net/npm/vue-router@3.1.6/dist/vue-router.min.js"
+          "vue-router": "https://cdn.jsdelivr.net/npm/vue-router@3.1.6/dist/vue-router.min.js",
```

Comment:
@tundealabi, please why are we importing vue and vue-router?

Reply:
dunno.. didn't add that

Reply:
I think it was @benjamin-bala, maybe for when a Vue app will be integrated...